src/BikramSambat.ts

```diff
@@ -22,4 +22,13 @@ export default class BikramSambat {
     this.month = month
     this.day = day
   }
+
+  public getPreviousYear(): number {
+    return this.year? - 1 :NaN
+  }
+   
+  public getNextYear(): number {
+    return this.year? + 1:NaN
```

Comment:
```suggestion
    return this.year ? +1 : NaN
```

Reply:
Prettier issue, (please check if prettier is properly setup to use project config or not)

Reply:
pretier was setup properly